.circleci/config.yml

```diff
@@ -952,19 +952,14 @@ jobs:
     system_tests:
         machine:
             docker_layer_caching: true
-            image: ubuntu-2004:202111-01
+            image: ubuntu-2204:current
```

Comment:
is `current` ok to use?

Reply:
From what I understand (CircleCI have a terrible documentation), this tag is for the most recent image (with security patches etc.), so I think this should be fine

Reply:
For testing purposes, that is. I wouldn't do it for releases

Reply:
https://discuss.circleci.com/t/new-image-tag-convention-for-ubuntu-linux-android-and-windows-vms/43750

Reply:
https://discuss.circleci.com/t/ubuntu-linux-vm-machine-images-2022-april-q2-update/43749